### PR TITLE
New script to Install Odoo 13 in Debian 10 Buster

### PR DESCRIPTION
**Pull request by rockcesar**
New script to Install Odoo 13 in Debian 10 Buster. Based in the script of Yenthe666.

Greetings.

### CODE REVIEW

File: odoo13_install_debian10.sh [hunk 1]
```diff
@@ -0,0 +1,359 @@
+#!/bin/bash
+################################################################################
+# Script for installing Odoo on Debian 10.0 (could be used for other version too)
+# Author: Yenthe Van Ginneken
+# Maintainer: César Cordero Rodríguez
```
Comment (Yenthe666) [inline odoo13_install_debian10.sh]: Can you set both of our names on both the author and maintainer lines please? Looks more correct then

**Comment by Yenthe666** [comment]
@rockcesar can you rename the file from "odoo13_install_debian10.sh" to "odoo_install_debian.sh"?

File: odoo13_install_debian10.sh (hunk 1 above)
Comment (rockcesar) [reply odoo13_install_debian10.sh]: Done. Thanks.

**Comment by rockcesar** [comment]
Author and Maintainer updated.
Name of the file changed.

**Comment by Yenthe666** [comment]
Thanks a lot @rockcesar! Great to have this added 🎈

**Comment by rockcesar** [comment]
Great!.

**Pull request merged**